internal/module/services/version_manager/version_manager.go

```diff
@@ -104,6 +119,30 @@ func (s *VersionManagerService) BeginVersion(ctx context.Context, request *terra
 		return nil, CreateModuleVersionError
 	}
 
+	parsedVersion := versions.MustParseVersion(strings.ReplaceAll(request.Module.GetVersion(), "v", ""))
+
+	// Check if official version.
+	if parsedVersion.GreaterThan(versions.MustParseVersion("0.0.0")) {
```

Comment:
This can be made a file level variable so we only do the conversion once see other comment.